packages/run-script/scripts/sonarConfig.ts

```diff
@@ -41,18 +46,23 @@ function getPrHeadRef(pr: any) {
 function rewriteCoverageReports(context: IContext) {
     // Workaround for https://community.sonarsource.com/t/code-coverage-doesnt-work-with-github-action/16747
     if (context.ci.service as any !== "github") {
+        context.logger.warn("Unable to rewrite coverage reports for a CI Environment other than GitHub");
         return;
     }
     const sonarProps = properties.of("sonar-project.properties");
     const reportPaths = sonarProps.get("sonar.javascript.lcov.reportPaths");
     if (typeof reportPaths !== "string") {
+        context.logger.warn("Unable to find the property: 'sonar.javascript.lcov.reportPaths'");
         return;
     }
     context.logger.info("Fixing coverage paths for SonarCloud");
     const pattern = new RegExp(context.env.GITHUB_WORKSPACE, "g");
     for (const reportPath of reportPaths.split(",")) {
+        context.logger.debug("Report file: " + reportPath);
         const reportText = fs.readFileSync(reportPath, "utf-8");
+        context.logger.debug("Contents before:\n" + reportText);
```

Comment:
Not sure if we want to log the entire file here, even in debug file mode, since it can be thousands of lines long.